commerce_coordinator/apps/commercetools/tests/conftest.py

```diff
@@ -170,7 +172,12 @@ def gen_payment():
         amount_planned=4900,
         payment_method_info={},
         payment_status=PaymentState.PAID,
-        transactions=[gen_transaction(TransactionType.REFUND, 4900)],
+        transactions=[gen_transaction(TransactionType.REFUND, CTTypedMoney(
```

Comment:
I remember we updated this mock function with the CT `Money` type but mocking started creating issues in other cases where this mock result was used. How have we resolved that?

Reply:
I didn't get that issue. I think that there must have been something wrong locally (maybe some unintentional code change) at that time.